openmdao/core/problem.py

```diff
@@ -434,7 +509,7 @@ def _check_mpi(self, out_stream=sys.stdout):
         if under_mpirun(): # pragma: no cover
             parr = True
             # Indicate that there are no parallel systems if user is running under MPI
-            if MPI.COMM_WORLD.rank == 0:
+            if self._comm.rank == 0:
```

Comment:
All references specific to a Problem have been replaced... there are other references (e.g. in utils, tests and dumprecorder) that did not require replacing.